bundles/org.openhab.binding.octopusenergy/src/main/resources/OH-INF/thing/things.xml

```diff
@@ -0,0 +1,219 @@
+		<item-type>DateTime</item-type>
+		<label>Last Refresh Time</label>
+		<description>The time consumption and tariff information was last refreshed</description>
+		<state readOnly="true" pattern="%1$tF %1$tR"/>
+	</channel-type>
+
+	<channel-type id="mpanType">
+		<item-type>String</item-type>
+		<label>MPAN</label>
+		<description>Meter Point Administration Number</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+
+	<channel-type id="mprnType">
+		<item-type>String</item-type>
+		<label>MPRN</label>
+		<description>Meter Point Reference Number</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+
+	<channel-type id="tariffType">
+		<item-type>String</item-type>
+		<label>Current Tariff</label>
+		<description>The currently applicable tariff</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+
+	<channel-type id="consumptionAmountType">
+		<item-type>Number:Energy</item-type>
+		<label>Consumed Energy Amount</label>
+		<description>The amount of energy consumed during the given time window</description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="consumptionStartTimeType">
+		<item-type>DateTime</item-type>
+		<label>Consumption Window Start Time</label>
+		<description>The start time of the most recent consumption window</description>
+		<state readOnly="true" pattern="%1$tF %1$tR"/>
+	</channel-type>
+
+	<channel-type id="consumptionEndTimeType">
+		<item-type>DateTime</item-type>
+		<label>Consumption Window End Time</label>
+		<description>The end time of the most recent consumption window</description>
+		<state readOnly="true" pattern="%1$tF %1$tR"/>
+	</channel-type>
+
+	<channel-type id="unitPriceWindowStartTimeType">
+		<item-type>DateTime</item-type>
+		<label>Unit Price Window Start Time</label>
+		<description>The start time of the unit price window</description>
+		<state readOnly="true" pattern="%1$tF %1$tR"/>
+	</channel-type>
+
+	<channel-type id="unitPriceWindowEndTimeType">
+		<item-type>DateTime</item-type>
+		<label>Unit Price Window End Time</label>
+		<description>The end time of the unit price window</description>
+		<state readOnly="true" pattern="%1$tF %1$tR"/>
+	</channel-type>
+
+	<channel-type id="maxUnitPriceType">
+		<item-type>Number</item-type>
+		<label>Maximum Unit Price (p/kWh)</label>
+		<description>The maximum unit price of energy in GBp per Kilowatt Hour over the available forecast period</description>
+		<state readOnly="true" pattern="%.1f"/>
+	</channel-type>
+
+	<channel-type id="minUnitPriceType">
+		<item-type>Number</item-type>
+		<label>Minimum Unit Price (p/kWh)</label>
+		<description>The minimum unit price of energy in GBp per Kilowatt Hour over the available forecast period</description>
+		<state readOnly="true" pattern="%.1f"/>
+	</channel-type>
+
+	<channel-type id="requestedStartTimeHourType">
+		<item-type>Number</item-type>
+		<label>Requested Activity Start Hour</label>
```

Comment:
This doesn't seem documented in the readme. Is this the result of the Action? What's the reason for writing it extra into a Channel?